README.md

```diff
@@ -1 +1,47 @@
-# ramen
+# Ramen
+
+Ramen is a **disaster-recovery orchestrator** for stateful applications across
+a set of peer kubernetes clusters, that are deployed and managed using
+[open-cluster-management](https://open-cluster-management.io/concepts/architecture/)
+(**OCM**), and provides **cloud-native interfaces** to orchestrate the
+life-cycle of an application's state. These include:
```

Comment:
Should we clarify what the application's state is?  Say by saying "to orchestrate the life-cycle of an application's state on PersistentVolumes"?